pandas/io/tests/test_parsers.py

```diff
@@ -2690,6 +2689,37 @@ def test_uneven_lines_with_usecols(self):
         df = self.read_csv(StringIO(csv), usecols=usecols)
         tm.assert_frame_equal(df, expected)
 
+    def test_read_empty_with_usecols(self):
```

Comment:
Thoughts on running this test under both `engine='c'` and `engine='python'? Are we that confident that any inconsistencies have already been worked out before we get here?